README.rst

```diff
@@ -0,0 +1,208 @@
+
+2. Analysis code
+----------------
+
+The analysis will consist of two stages. In the first stage, we shall process
+the original collision data (using ``demoanalyzer_cfg_level3data.py``) and
+simulated data (using ``demoanalyzer_cfg_level3MC.py``) for one Higgs signal
+candidate with with reduced statistics. In the second stage, we shall plot the
+results (using ``M4Lnormdatall_lvl3.cc``). The ``HiggsDemoAnalyzer`` directory
+contains the analysis code plugin for the `CMSSW <http://cms-sw.github.io/>`_
+analysis framework.
+
+3. Compute environment
+----------------------
+
+In order to be able to rerun the analysis even several years in the future, we
+need to "encapsulate the current compute environment", for example to freeze the
+software package versions our analysis is using. We shall achieve this by
+preparing a `Docker <https://www.docker.com/>`_ container image for our analysis
+steps.
+
+This analysis example runs within the `CMSSW <http://cms-sw.github.io/>`_
+analysis framework that was packaged for Docker in `clelange/cmssw
+<https://hub.docker.com/r/clelange/cmssw/>`_.
+
+4. Analysis workflow
+--------------------
+
+The analysis workflow is simple and consists of two above-mentioned stages:
+
+.. code-block:: console
+
+                              START
+                             /     \
+                            /       \
+                           /         \
+   +-------------------------+     +------------------------+
+   | process collision data  |     | process simulated data |
+   +-------------------------+     +------------------------+
+                   \                       /
+                    \ Higgs4L1file.root   / DoubleMuParked2012C_10000_Higgs.root
+                     \                   /
+                  +-------------------------+
+                  |    produce final plot   |
+                  +-------------------------+
+                             |
+                             | mass4l_combine_userlvl3.pdf
+                             V
+                            STOP
+
+We shall use the `CWL <http://www.commonwl.org/v1.0/>`_ workflow specification
+to express the computational workflow:
+
+- `workflow definition <workflow/workflow.cwl>`_
+
+and its individual steps:
+
+- `process collision data <workflow/step1data.cwl>`_
+- `process simulated data <workflow/step1mc.cwl>`_
+- `produce final plot <workflow/step2.cwl>`_
+
+5. Output results
+-----------------
+
+The example produces a plot showing the Higgs signal:
+
+.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-root6-roofit/master/docs/mass4l_combine_userlvl3.png
+   :alt: mass4l_combine_userlvl3.png
+   :align: center
+
+Local testing
+=============
+
+*Optional*
+
+If you would like to test the analysis locally (i.e. outside of the REANA
+platform), you can proceed as follows.
+
+Using pure Docker:
+
+.. code-block:: console
+
+    $ docker run -i -t --rm \
```

Comment:
The indentation of the docker commands is a bit odd, but anyway, it's well readable.

Reply:
We can change this once we reanify the example